core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1911,7 +1911,7 @@ public View getPrimaryView() {
      }
 
     public void setPrimaryView(@Nonnull View v) {
-        checkPermission(Permission.CONFIGURE);
+        checkPermission(Jenkins.ADMINISTER);
```

Comment:
Is this in the JEP? that the primary view is only allowed to administer?

Reply:
no it's the last ongoing discussion on the PR https://github.com/jenkinsci/jenkins/pull/4374#discussion_r350830926
Anyway I think that Permission.CONFIGURE is a mistake, it should be Jenkins.ADMINISTER or Jenkins.MANAGE, but reading the discussion on the PR I think it's for ADMINISTER.